galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/ExtractQuerySort.java

```diff
@@ -38,7 +38,7 @@
     }
 	
 	public static Boolean isAscending(Map<String, String[]> query, String param) {
-		return extractParameters(query).get(param);
+			return extractParameters(query).get(param);
```

Comment:
This isn't right is it?    The default is true.    if the parameter is missing,  how does this work?

Reply:
extractParameters looks for "desc" if it doesnt find that it just returns TRUE. In RequestorRas the list is always sorted in asc as default

Reply:
I still dont believe this is right.
Pass ?sort=bob:asc
and call the function with "jim" and see what happens